templates/masterclasses.html

```diff
@@ -141,41 +146,52 @@ <h3 class="p-muted-heading u-sv2">Something missing?</h3>
       }
     });
   });
-  resetSearch.addEventListener('click', () => {
-    cards.forEach((card) => {
-      card.style.display = 'block';
-    });
-  });
+  resetSearch.addEventListener('click', resetCards);
   // Prevent form submission
   submitButton.addEventListener('click', (e) => {
     e.preventDefault();
   })
-  Array.from(document.getElementsByClassName("p-chip")).forEach((el) => el.addEventListener("click", function () {
+
+  const changeTag = (node, tag) => {
+    const clone = document.createElement(tag)
+    for (const attr of node.attributes) {
+      clone.setAttributeNS(null, attr.name, attr.value)
+    }
```

Comment:
Maybe we can set attribute like this, without specifying a namespace?

```suggestion
    for (const { name, value } of node.attributes) {
      clone.setAttribute(name, value);
    }
```